backend/lib/services/cloudprofiles.js

```diff
@@ -79,15 +84,16 @@ exports.list = function () {
     .value()
 }
 
-exports.read = function ({ name }) {
+exports.read = async function ({ name, user }) {
   const cloudProfiles = getCloudProfiles()
   const cloudProfileResource = _.find(cloudProfiles, ['metadata.name', name])
   if (!cloudProfileResource) {
     throw new NotFound(`Cloud profile with name ${name} not found`)
   }
 
   const seeds = getVisibleAndNotProtectedSeeds()
-  const cloudProfile = assignSeedsToCloudProfileIteratee(seeds)(cloudProfileResource)
+  const { items: gardenShoots } = await shoots.list({ user, namespace: 'garden' })
```

Comment:
```suggestion
  const client = user.client
  const { items: gardenShoots } = await client['core.gardener.cloud'].shoots.list('garden')
```

Reply:
If the user has no authorization for ns garden?

Reply:
see comment above